[2022-02-15 03:25:00,675] {taskinstance.py:1032} INFO - Dependencies all met for <TaskInstance: dbt_training_bytecode.model.my_new_project.fct_orders manual__2022-02-15T03:24:14.594120+00:00 [queued]>
[2022-02-15 03:25:00,704] {taskinstance.py:1032} INFO - Dependencies all met for <TaskInstance: dbt_training_bytecode.model.my_new_project.fct_orders manual__2022-02-15T03:24:14.594120+00:00 [queued]>
[2022-02-15 03:25:00,705] {taskinstance.py:1238} INFO - 
--------------------------------------------------------------------------------
[2022-02-15 03:25:00,707] {taskinstance.py:1239} INFO - Starting attempt 1 of 1
[2022-02-15 03:25:00,708] {taskinstance.py:1240} INFO - 
--------------------------------------------------------------------------------
[2022-02-15 03:25:00,729] {taskinstance.py:1259} INFO - Executing <Task(BashOperator): model.my_new_project.fct_orders> on 2022-02-15 03:24:14.594120+00:00
[2022-02-15 03:25:00,740] {standard_task_runner.py:52} INFO - Started process 2100 to run task
[2022-02-15 03:25:00,750] {standard_task_runner.py:76} INFO - Running: ['***', 'tasks', 'run', 'dbt_training_bytecode', 'model.my_new_project.fct_orders', 'manual__2022-02-15T03:24:14.594120+00:00', '--job-id', '132', '--raw', '--subdir', 'DAGS_FOLDER/dbt-***.py', '--cfg-path', '/tmp/tmpf8kpzukl', '--error-file', '/tmp/tmpnqx9dh4l']
[2022-02-15 03:25:00,755] {standard_task_runner.py:77} INFO - Job 132: Subtask model.my_new_project.fct_orders
[2022-02-15 03:25:00,962] {logging_mixin.py:109} INFO - Running <TaskInstance: dbt_training_bytecode.model.my_new_project.fct_orders manual__2022-02-15T03:24:14.594120+00:00 [running]> on host 459d09dd7c86
[2022-02-15 03:25:01,094] {taskinstance.py:1426} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=Bytecode
AIRFLOW_CTX_DAG_ID=dbt_training_bytecode
AIRFLOW_CTX_TASK_ID=model.my_new_project.fct_orders
AIRFLOW_CTX_EXECUTION_DATE=2022-02-15T03:24:14.594120+00:00
AIRFLOW_CTX_DAG_RUN_ID=manual__2022-02-15T03:24:14.594120+00:00
[2022-02-15 03:25:01,096] {subprocess.py:62} INFO - Tmp dir root location: 
 /tmp
[2022-02-15 03:25:01,100] {subprocess.py:74} INFO - Running command: ['bash', '-c', '\n            dbt --no-write-json run --target dev --models fct_orders             --profiles-dir /opt/***/dbt/dbt-training-bytecode-1 --project-dir /opt/***/dbt/dbt-training-bytecode-1\n            ']
[2022-02-15 03:25:01,127] {subprocess.py:85} INFO - Output:
[2022-02-15 03:25:09,602] {subprocess.py:89} INFO - 03:25:09  Running with dbt=1.0.1
[2022-02-15 03:25:10,018] {subprocess.py:89} INFO - 03:25:10  Found 8 models, 11 tests, 0 snapshots, 0 analyses, 367 macros, 0 operations, 1 seed file, 3 sources, 0 exposures, 0 metrics
[2022-02-15 03:25:10,025] {subprocess.py:89} INFO - 03:25:10
[2022-02-15 03:25:14,004] {subprocess.py:89} INFO - 03:25:14  Concurrency: 4 threads (target='dev')
[2022-02-15 03:25:14,009] {subprocess.py:89} INFO - 03:25:14
[2022-02-15 03:25:14,047] {subprocess.py:89} INFO - 03:25:14  1 of 1 START table model dbt_bytecode_training.fct_orders....................... [RUN]
[2022-02-15 03:25:19,410] {subprocess.py:89} INFO - 03:25:19  1 of 1 OK created table model dbt_bytecode_training.fct_orders.................. [SUCCESS 1 in 5.35s]
[2022-02-15 03:25:19,529] {subprocess.py:89} INFO - 03:25:19
[2022-02-15 03:25:19,544] {subprocess.py:89} INFO - 03:25:19  Finished running 1 table model in 9.49s.
[2022-02-15 03:25:19,547] {subprocess.py:89} INFO - 03:25:19
[2022-02-15 03:25:19,549] {subprocess.py:89} INFO - 03:25:19  Completed successfully
[2022-02-15 03:25:19,551] {subprocess.py:89} INFO - 03:25:19
[2022-02-15 03:25:19,554] {subprocess.py:89} INFO - 03:25:19  Done. PASS=1 WARN=0 ERROR=0 SKIP=0 TOTAL=1
[2022-02-15 03:25:20,207] {subprocess.py:93} INFO - Command exited with return code 0
[2022-02-15 03:25:20,281] {taskinstance.py:1277} INFO - Marking task as SUCCESS. dag_id=dbt_training_bytecode, task_id=model.my_new_project.fct_orders, execution_date=20220215T032414, start_date=20220215T032500, end_date=20220215T032520
[2022-02-15 03:25:20,401] {local_task_job.py:154} INFO - Task exited with return code 0
[2022-02-15 03:25:20,578] {local_task_job.py:264} INFO - 0 downstream tasks scheduled from follow-on schedule check
